gdraw/gresource.c

```diff
@@ -788,7 +784,7 @@ static void _GResourceFindImage(const char *fname, GResImage *img) {
     if ( fname!=NULL ) {
 	t = _GGadgetImageCache(fname, false);
 	if ( t==NULL || t->image==NULL ) {
-	    GDrawError("Could not find image corresponding to '%s', using default", fname);
```

Comment:
This was pretty noisy when I was running, I think it makes sense to move these to traces

Reply:
I'm fine with that. Want me to do it?

Reply:
Ah yeah, I've already moved them to traces